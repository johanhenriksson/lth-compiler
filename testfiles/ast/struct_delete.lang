struct s { }

void main() {
    s t;
    delete t;
}
