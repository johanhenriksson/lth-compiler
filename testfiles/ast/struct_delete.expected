Program 
  List 2 item(s)
    Struct s
      TypeDecl s
      List 0 item(s)
    Function main()
      IdDecl void main
        TypeId void
      List 0 item(s)
      Block body
        List 2 item(s)
          DeclareStmt s t;
            IdDecl s t
              TypeId s
            Opt ?
          DeleteStmt t
            IdUse t
